### DISCUSSION TITLE
Bash setup.sh error - Elasticsearch not starting

### DISCUSSION THREAD

**Discussion by euanmillar** [opencrvs/opencrvs-core#4272] [Technical Q&A] [answered]
We have been notified that recently, the `bash setup.sh` script has failed for some users at this step.  It appears that Elasticsearch is not starting.  

![image](https://user-images.githubusercontent.com/3169954/201656683-cd068a6f-86cc-4f12-80b1-b96b4eccefc9.png)

**Comment by euanmillar** [comment]
Please subscribe to this ticket for a resolution: https://github.com/opencrvs/opencrvs-core/issues/4273

**Comment by euanmillar** [comment]
![euan-ubuntu](https://user-images.githubusercontent.com/3169954/201665072-7fa19314-63a9-4973-9ee8-c50a8e1c5615.png)

At the moment, this is working for us.  We are working with users who are experiencing this issue to find out more information.

**Comment by euanmillar** [comment, marked as answer]
After some investigation, we discovered that users are either:

1. Cloning the repository with **sudo** and then using `sudo bash setup.sh` which is a mistake.  **Do not use "sudo"!**  ... or
2. Trying to run the `bash setup.sh` command on an external server.

**The bash setup.sh command is intended to help you run OpenCRVS locally, ie on your own personal laptop.**

- If you have Ubuntu installed on your laptop, this is best and you can proceed to use the bash setup command.
- If you are on Windows, you can run a Ubuntu virtual machine using Virtual Box and run bash setup.sh in there.
- If you are on a Mac, OpenCRVS is Unix compatible.  Simply install Docker for Mac and all the other dependencies listed [here](https://documentation.opencrvs.org/setup/3.-installation/3.1-set-up-a-development-environment/3.1.1-install-the-required-dependencies) then continue with the [setup commands](https://documentation.opencrvs.org/setup/3.-installation/3.1-set-up-a-development-environment/3.1.2-install-opencrvs-locally).

**Reply by euanmillar** [reply]
Please use a fresh Ubuntu image with default permissions.  You do not need to change any permissions on your default Ubuntu installation.

**Reply by papouso** [reply]
Thank you euanmillar for providing tracks to solve this issue. I am facing this issue for a while now and i am very frustrated i can't successfully install OpenCRVS as we envison to deploy it.
As your 2nd recommandations, you suggest to use a laptop and install OpenCRVS locally. 
The point is that in the system requirements, it's said 16Gb of RAM and most Laptop don't have such RAM (at least mine) and in the otherhand how a system that is entend to be used at national level can be advised to be install on local laptop and not on a full flaged cloud server(since the end of the day the deployment will be done in a server whatever barematal or cloud).
And also ragarding the system requirement is it Ubuntu desktop or Ubuntu server and also which distribution (20 or 22 or 23). 
I will really appreciate if the installation process can be made more clear regarding test or production environnement and the exact requirement like hardware and software to avoid depencance issue (i also met such issues with node).
I am sure this will help us as integrators to rapidly adopt the platform and spread it as there is really a need of it and we are very interested about it.
Thank you in advance for your support.
